src/ui/public/styles/dark-theme.less

```diff
@@ -313,9 +313,28 @@
 
 // /src/ui/public/vislib/styles/_legend.less
```

Comment:
The reason I had this code segmented by source file was so that the two could stay in sync, and any tweaks to colors would be made in the dark-variables.less file only.